file-proxy/src/main/resources/application.conf

```diff
@@ -0,0 +1,61 @@
+akka {
+  log-level = INFO
+  http.server.preview.enable-http2 = on
+}
+
+akka.http.parsing {
+  max-to-strict-bytes = 20m
+}
+akka.http.server.parsing {
+  max-content-length = 20m
+}
+
+akka.http.server.request-timeout = 4888888 seconds
```

Comment:
Let's try setting this to `infinite`  to see if we can get around just setting some arbitrarily large value  ( I know we're doing this in other places in the code it'll be good to test it out here).